addons/tutorial_04_split_tunneling/vpnIsOn.js

```diff
@@ -1,22 +1,17 @@
-// Let's use a global variable to keep track of the split-tunneling settings.
-// This will be replaced with addon properties as soon as they will be
-// implemented (v2.13~).
-splitTunnelTutorialVpnWasOn = false;
+(function ({ connectSignal, controller, addon }, condition) {
+  connectSignal(addon, 'playingChanged', () => {
+    if (!addon.playing) {
+      return;
+    }
 
-(function(api, condition) {
-api.connectSignal(api.addon, 'playingChanged', () => {
-  if (!api.addon.playing) {
-    return;
-  }
+    if (controller.state === controller.StateOff) {
+      condition.disable();
+      addon.state.session.set("splitTunnelTutorialVpnWasOn", false);
```

Comment:
Let's make this file back-compatible. Maybe with `if ("state" in addon) ...`

Reply:
We shouldn't merge this actually... I left this for illustration purposes, I am going to drop this commit before merging. This tutorial is disabled starting from v2.14, so I think there is no point.

Reply:
Already discussed this offline with @bakulf and he agrees with the above comment :)